data-prepper-plugins/grok-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/grok/GrokProcessorConfig.java

```diff
@@ -42,8 +42,8 @@ public class GrokProcessorConfig {
     static final String DEFAULT_TARGET_KEY = null;
 
     @JsonProperty(BREAK_ON_MATCH)
-    @JsonPropertyDescription("Specifies under what condition the `grok` processor should perform matching. " +
-            "Default is no condition.")
+    @JsonPropertyDescription("Specifies whether to match all patterns (`true`) or stop once the first successful " +
```

Comment:
It's actually swapped here. The `true` specifies to stop once the first successful match is found, and false matches all pattern